src/Nethermind/Nethermind.Init/Steps/InitializeNetwork.cs

```diff
@@ -203,8 +203,11 @@ await StartDiscovery().ContinueWith(initDiscoveryTask =>
             throw new InvalidOperationException("Cannot initialize network without knowing own enode");
         }
 
+        ProductInfo.InitializePublicClientId(_networkConfig.PublicClientIdFormat);
+
         ThisNodeInfo.AddInfo("Ethereum     :", $"tcp://{_api.Enode.HostIp}:{_api.Enode.Port}");
         ThisNodeInfo.AddInfo("Client id    :", ProductInfo.ClientId);
+        ThisNodeInfo.AddInfo("Public client id    :", ProductInfo.PublicClientId);
```

Comment:
Is this nicely aligned?

Reply:
Changed it to "Public id" to avoid having to make changes in all the other places where AddInfo is used, should be explicit enough. If not i'll make the changes to align everything.